grabdish/config/db/2db-sa/apply/03-db1-aq.sql

```diff
@@ -0,0 +1,29 @@
+-- Copyright (c) 2021 Oracle and/or its affiliates.
```

Comment:
could these file names be more descriptive? eg this file seems to grant privs and add subscriber but is named grabdish/config/db/2db-sa/apply/03-db1-aq.sql

Reply:
The names are designed to be processed automatically.  The number, for example 03, defines the order of application.  db1 means apply to db1, the aq refers to AQ settings.  I will add a little more description to the name.